doc/setup.md

```diff
@@ -216,3 +220,27 @@ pg1*/status.sh
 mc1*/status.sh
 mc1*/backstage/status.sh
 ```
+
+### Run tests from [/t/environs](/t/environs) with docker, manually for debugging
+- Requires docker configured for non-root users
+- Available configuration:
+  - `MIRRORCACHE_CITY_MMDB=/var/lib/GeoIP/GeoLite2-City.mmdb` adds this environment variable inside the container and mounts it as a volume if the file exists on the host
+    ```bash
+    cd t/environs
+    # Just run the test:
+    ./01-smoke.sh
+    # Run the test with your own MIRRORCACHE_CITY_MMDB
+    MIRRORCACHE_CITY_MMDB=/var/lib/GeoIP/GeoLite2-City.mmdb ./01-smoke.sh
+    ```
+  - `EXPOSE_PORT=3190` maps port 3190 or whatever port you need from the container to host port 80, but also requires one more change, to ensure that MirrorCache is listening to requests from the host:
+    ```bash
+    # Add this line before mc9*/start.sh in your test
+    sed -i 's,MOJO_LISTEN=http://127.0.0.1,MOJO_LISTEN=http://*,' mc9*/start.sh
```

Comment:
I think this may be addressed in environs project, I will work on it, then we will remove this from the doc when it is ready.

Reply:
Sounds good. Morbo reloading, too.